libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/SessionCredentialItem.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.di.ipv.core.library.persistence.item;
+
+import com.nimbusds.jwt.SignedJWT;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbSortKey;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@DynamoDbBean
```

Comment:
Do we need this to include `@Getter` (or `@Data`) so we can actually read the credential...

Reply:
As above - intentially left off as not needed right now. Will add them as required.